src/components/MemberCard/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Card, Row, Avatar } from "antd";
+import { LinkedinFilled, GithubFilled } from "@ant-design/icons";
+import "./index.css";
+
+// Props passed to this function are details of team members
+const MemberCard = ({
+  // Member's name
```

Comment:
actually these props are self-explanatory. unlike the case of @tmansuroglu puzzle page or your game for example where the props are ambiguous.

so, you can delete all these comments

and then you rename
```
avatar -> avatarUrl
linkedin -> linkedinUrl
github -> githubUrl
```

and you don't need any comments. the variable names are self documenting.

Reply:
PTAL